lib/ansible/cli/inventory.py

```diff
@@ -9,6 +9,7 @@
 
 import argparse
 from operator import attrgetter
+from hashlib import sha1
```

Comment:
I guess this can be removed. You import sha1 locally below.

Reply:
fixed in 7b1e546